rim/templates/rim/client.html

```diff
@@ -0,0 +1,80 @@
+{% extends "rim/base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block content %}
+
+    <div class='grid-x grid-padding-x'>
+      <div class='small-12 cell'>
+        <h2>{{ client.name }}</h2>
+      </div>
+      <div class='small-3 cell'>
+        <h4>{{ client.bpn }}</h4>
+      </div>
+    </div>
+    <div class='grid-x grid-padding-x'>
+      <div class='small-12 cell'>
+        <h2>{% trans 'Current Checkout' %}</h2>
```

Comment:
Maybe change this to Current Checkout**s**.

Also, you should center it so it's easier to differentiate between this title and the Client's name/BPN (use [Foundation's "text-center" helper class](https://foundation.zurb.com/sites/docs/v/5.5.3/utility-classes.html))

Reply:
fixed